activesupport/lib/active_support/test_case.rb

```diff
@@ -12,6 +12,7 @@
 require "active_support/testing/time_helpers"
 require "active_support/testing/file_fixtures"
 require "active_support/testing/parallelization"
+require "concurrent"
```

Comment:
Do we need to require the whole thing here?

Reply:
Seems like a good idea just to require `"concurrent/utility/processor_counter"`